data_labeling/DeepCell_Label_interactive_demo.md

```diff
@@ -0,0 +1,351 @@
+
+### Changing the displayed image slice
+
+#### Change frames
+Press the "d" or right arrow key to go forward through frames.
+
+Press the "a" or left arrow key to go backward through frames.
+
+You can cycle to the last/first frame when viewing the first/last frame.
+
+#### Change channel
+Press "C" or "Shift+C" to change the channel of the displayed image. "C" increments the channel, while "Shift+C" decrements the channel.
+
+Make sure you have no selected labels when changing channels, as the "c" key can change labels in whole-label mode.
+
+Brightness and contrast adjustments for each channel are stored seperately. Try adjusting the brightness and contrast in a few channels and see what it's like to cycle between them!
+
+#### Change feature
+Press "F" key (to increment) or "Shift+F" (to decrement) to change the displayed feature. 
+
+Make sure you have no labels selected, as "f" fills holes in whole-label mode.
+
+The demo file only contains one feature, so you will not be able to try this functionality right now.
+
+Now that you know how to move through and view files with DeepCell Label, take a minute to explore the file. Next, we will cover whole-label actions starting in frame 5 of the demo file, so make sure you can get to this frame and toggle whole-label mode using the controls covered so far.
+
+## Whole-label mode
+In this section, we will cover the whole-label actions available to edit labels. We will use these actions to fix specific errors throughout the label file, but you can also try them on other labels to see how they work.
+
+### Select labels in whole-label mode
+Click on a label to select it. We can select up to two labels in whole-label mode. Some actions require just one selected label, while others require two selected labels. Clicking on a label after selecting two overwrites the second selected label with the new label.
+
+#### Highlight labels in whole-label mode
+In whole-label mode, we can highlight the selected labels. Highlighting makes selected labels bright red. We recommend using highlighting to make sure we have selected the label(s) we intend to modify.
+
+If we've only selected one label, we can cycle the highlighted label with the bracket keys ( "\[" and "\]" ). This deselects the original selected label, but keeps the highlight active.
+
+### Swap labels
+
+Select two different labels, then press the "s" key to swap the selected labels. A prompt will appear in "state" to confirm the swap where we can
+ - press "S" (again) to swap the labels in only the current frame,
+ - press the spacebar to swap them across all frames, or
+ - press "Esc" to abort the action.
+
+This can be useful in timelapse or 3D files with many frames, where consistent labels across each frame are required for labeling correctness.
+
+#### Suggested swap
+![DeepCell Label demo labels for suggested swap](https://figure-eight-deepcell.s3.us-east-2.amazonaws.com/instructions_and_examples/janelia_demo/resized_suggested_swap.png)
```

Comment:
```suggestion
Select two different labels, then press the "s" key to swap the selected labels. A prompt will appear in "state" to confirm the swap where we can:

- press "S" (again) to swap the labels in only the current frame,
- press the spacebar to swap them across all frames, or
- press "Esc" to abort the action.

This can be useful in timelapse or 3D files with many frames, where consistent labels across each frame are required for labeling correctness.

#### Suggested swap

![DeepCell Label demo labels for suggested swap](https://figure-eight-deepcell.s3.us-east-2.amazonaws.com/instructions_and_examples/janelia_demo/resized_suggested_swap.png)
```